source/puppet/5.3/hiera_config_yaml_5.md

```diff
@@ -64,7 +64,9 @@ hierarchy:
       - 'common.yaml'
 ```
 
-Options for tweaking the hiera.yaml file:
+Note that `data_hash: yaml_data` and `datadir: data` are the default settings.
```

Comment:
I find this confusing; are they like "factory default", or did the example above define them as defaults? If so, how did the use of `data_hash: yaml_data` make that default, it is not in the `defaults`. 

If those lines were commented out it would make sense.
```
--
 version: 5

# defaults:
#   datadir: "data"
 		 
hierarchy:
   - name: 'Yaml backend'
#     data_hash: yaml_data
     paths:	
       - "nodes/%{trusted.certname}.yaml"		       
        - 'common.yaml'
  ```